lighthouse-core/scripts/publish-prepare-pristine.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT="$DIRNAME/../.."
+cd $LH_ROOT
+
+set -euxo pipefail
+
+# Setup a pristine git environment
+cd ../
+
+if [[ ! -e lighthouse-pristine/ ]]; then
+  git clone git@github.com:GoogleChrome/lighthouse.git lighthouse-pristine
+fi
+
+cd lighthouse-pristine/
+
+if [[ -n "$(git status --porcelain)" ]]; then
+  echo "Pristine repo has changes to the files! Commit or stash the changes to continue."
+  exit 1
+fi
+
+git fetch origin
+git fetch --tags
+git checkout -f master
+git reset --hard origin/master
+git clean -fx
```

Comment:
/me reads `git help clean`.

wow. i'm so pleased that they use the term `pristine` as well. woo! naming win. \o/

I usually see `-d` hanging out along with `-f`. I didn't know about `-x` but looks like a good call.

Reply:
SGTM! `-d` added 👍